### ISSUE TITLE
OSError: Search pattern never detected in send_command_expect: \(tcl\)#

### ISSUE THREAD

**Issue by VDavidCC** [napalm-automation/napalm#1234]
Hi!
Im having this error when i try to apply commit.changes:
OSError: Search pattern never detected in send_command_expect: \(tcl\)#

This is my script:
`from napalm import get_network_driver

with open('Lista.txt') as f:
    IpList = f.read().splitlines()

for ip_address in IpList:
    print ("Connecting to " + str(ip_address))
    driver = get_network_driver('ios')
    iosv = driver(ip_address, 'user', 'pass', optional_args= {'secret': 'pass','inline_transfer': 'True','session_log': 'my_file.txt'})
    iosv.open()
    print("Conexion exitosa!")
    iosv.load_merge_candidate(filename='commands.txt')
    iosv.commit_config()
    print ("El equipo fue configurado!")`
And this is the content of commands.txt:
`snmp-server community test050620`

Network Operation System:
It is a Cisco Access Point C1140
Cisco IOS Version 12.4(23c)JA2

In the session_log i found this:
`MEX1MAYP1.AP03>
MEX1MAYP1.AP03>terminal length 0
MEX1MAYP1.AP03>terminal width 511
MEX1MAYP1.AP03>
MEX1MAYP1.AP03>enable
Password: 
MEX1MAYP1.AP03#
MEX1MAYP1.AP03#
MEX1MAYP1.AP03#
MEX1MAYP1.AP03#dir
Directory of flash:/

    2  -rwx       95705   Mar 2 2020 22:41:50 +00:00  event.log
    3  drwx           0   Mar 1 1993 00:00:33 +00:00  configs
    4  -rwx       34076   Feb 9 2018 19:57:08 +00:00  lwapp_non_apspecific_reap.cfg
   44  drwx         384   May 9 2019 16:12:42 +00:00  c1140-k9w8-mx.124-23c.JA2
    5  -rwx        8216   Jun 4 2020 21:47:01 +00:00  private-multiple-fs
    7  drwx         128   Mar 1 2002 00:12:36 +00:00  c1140-rcvk9w8-mx
    6  -rwx         280  May 11 2020 13:48:56 +00:00  lwapp_officeextend.cfg
   12  -rwx        3566   Jun 4 2020 16:27:03 +00:00  private-config
   11  -rwx       27576   Jun 4 2020 21:47:01 +00:00  lwapp_reap.cfg
   41  -rwx       82136  Jan 24 2018 22:08:54 +00:00  event.capwap
   13  -rwx       27576   May 9 2019 16:03:32 +00:00  lwapp_reap.cfg.bak
   14  -rwx         363  May 11 2020 13:48:56 +00:00  env_vars

32126976 bytes total (24506368 bytes free)
MEX1MAYP1.AP03#
MEX1MAYP1.AP03#dir flash:
Directory of flash:/

    2  -rwx       95705   Mar 2 2020 22:41:50 +00:00  event.log
    3  drwx           0   Mar 1 1993 00:00:33 +00:00  configs
    4  -rwx       34076   Feb 9 2018 19:57:08 +00:00  lwapp_non_apspecific_reap.cfg
   44  drwx         384   May 9 2019 16:12:42 +00:00  c1140-k9w8-mx.124-23c.JA2
    5  -rwx        8216   Jun 4 2020 21:47:01 +00:00  private-multiple-fs
    7  drwx         128   Mar 1 2002 00:12:36 +00:00  c1140-rcvk9w8-mx
    6  -rwx         280  May 11 2020 13:48:56 +00:00  lwapp_officeextend.cfg
   12  -rwx        3566   Jun 4 2020 16:27:03 +00:00  private-config
   11  -rwx       27576   Jun 4 2020 21:47:01 +00:00  lwapp_reap.cfg
   41  -rwx       82136  Jan 24 2018 22:08:54 +00:00  event.capwap
   13  -rwx       27576   May 9 2019 16:03:32 +00:00  lwapp_reap.cfg.bak
   14  -rwx         363  May 11 2020 13:48:56 +00:00  env_vars

32126976 bytes total (24506368 bytes free)
MEX1MAYP1.AP03#tclsh
Translating "tclsh"...domain server (255.255.255.255)
% Unknown command or computer name, or unable to find computer address
MEX1MAYP1.AP03#`

I have tried manually typing the command and i have no problems nor slow-feel...

**Comment by ktbyers** [comment]
Get rid of:

```
'inline_transfer': 'True'
```

You will need to enable Secure Copy on the AP however (so it must support that and have it enabled).

**Comment by VDavidCC** [comment]
Well in my case i would have to apply the Scp server enable command manually on many APs (around 200) so i preferred to use Netmiko to directly apply commands on configuration mode.
Thank you!

**Comment by ktbyers** [comment]
Yes, understood...use Netmiko to enable SCP (if you want to use NAPALM's merge operation).
